src/asymmetric_encryption.cpp

```diff
@@ -96,6 +100,11 @@ std::vector<uint8_t> AsymmetricEncryption::encrypt(AsymmetricPublicKey key,
         }
 
         int maxSize{pad.getDataMaxSize(key)};
+        /* Validate message size when not using padding*/
```

Comment:
Please split this into a separate commit. I would like to have one commit per feature.

Reply:
Done.